### PR TITLE
SC membership changes

### PR DESCRIPTION
**Pull request by sashakames**
updates to the lists and removing duplicate info.

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -70,32 +70,5 @@ Meetings may occur via telecom, e-mail, or face-to-face.  The XC will hold regul
...
-  * Cecilia DeLuca (NOAA)
-  * V. Balaji (NOAA)
-  * Ben Evans (NCI)
-  * Claire Trenham (NCI)
-
-
-[download]({{site.esgf-media}}/pdf/ESGF_Governance_6_4_2015.pdf)
\ No newline at end of file
+
+[download]({{site.esgf-media}}/pdf/ESGF_Governance_6_4_2015.pdf)
```
Comment (mauzey1) [inline governance.md:74]:
```suggestion
[download]({{site.esgf-media}}/pdf/ESGF_Governance_5_11_2017.pdf)
```
Should we update this link?

**Review by mauzey1 — CHANGES_REQUESTED** [review]

**Comment by mauzey1** [comment]
@sashakames It seems that no place on the website links to the governance page.  The `Governance` link under the documents navigation menu goes to a [PDF file](https://esgf.llnl.gov/esgf-media/pdf/ESGF_Governance_5_11_2017.pdf).  Should that link go to the governance page instead?

**Comment by sashakames** [comment]
Thank you for catching that. I saw the lists on that page but didn't realize it doesn't link.  Perhaps best to just delete the file then.

**Comment by mauzey1** [comment]
Wouldn't it be better to keep the page since it can be updated instead of linking to a document from 2017?

**Comment by sashakames** [comment]
Good question Chris.  Lets keep things simple/as is for now.   The website has needed a major overhaul for a very long time, but out of scope for our work.

**Review by mauzey1 — APPROVED** [review]

**Pull request merged**